src/ServiceContainer/WebApiExtension.php

```diff
@@ -50,7 +50,7 @@ public function configure(ArrayNodeDefinition $builder)
         $builder
             ->addDefaultsIfNotSet()
             ->children()
-                ->scalarNode('base_url')
+                ->scalarNode('base_uri')
```

Comment:
well, this is our own configuration. It does not need to match the Guzzle setting